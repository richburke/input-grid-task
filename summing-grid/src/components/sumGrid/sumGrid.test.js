import React from 'react';
import { shallow } from 'enzyme';
import SumGrid from './';
import Cell from '../cell';

describe('SumGrid', () => {
  it('should render correctly', () => {
    const component = shallow(<SumGrid />);
    expect(component).toMatchSnapshot();
  });

  it('should render its container <div>', () => {
    const component = shallow(<SumGrid />);
    expect(component.find('div.grid')).toBeDefined();
  });

  it('should render at least one Cell', () => {
    const component = shallow(<SumGrid />);
    expect(component.find(Cell)).toBeDefined();
  });
});
