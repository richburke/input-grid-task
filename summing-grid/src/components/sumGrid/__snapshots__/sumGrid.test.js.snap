// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SumGrid should render correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
